### PR TITLE
Fix work of websockets

### LINKED ISSUE #11: Fix work of websockets

### PR DESCRIPTION
**Pull request by CedrikNikita**
fixes #11

### CODE REVIEW

File: store/index.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
-import axios from 'axios'
+import camelcaseKeysDeep from 'camelcase-keys-deep'
+import { initMiddleware } from './utils'
 
 export const state = () => ({
   nodeStatus: {},
+  middleware: null,
```
Comment (davidyuk) [inline store/index.js:6]:
```suggestion
  middleware: initMiddleware(),
```
🤷‍♀️

File: store/utils.js [hunk 2]
```diff
@@ -1,3 +1,79 @@
...
+
+  const swg = Swagger.compose({
+    methods: {
+      urlFor: (path) => process.env.middlewareURL + path,
+      axiosError: () => ''
+    }
+  })({ swag })
+  swg.Swagger = { defaults: {} }
+
+  return camelcaseKeysDeep(swg.api)
```
Comment (davidyuk) [inline store/utils.js:74]: This is fine, but I'm going to fix this on the SDK side https://github.com/aeternity/aepp-sdk-js/issues/1119

File: store/utils.js [hunk 3]
```diff
@@ -1,3 +1,79 @@
...
+          {
+            in: 'path',
+            name: 'name',
+            required: true,
+            type: 'string'
+          }
+        ]
+      }
+    },
+    'txs/backward': {
```
Comment (davidyuk) [inline store/utils.js:21]: Please, ensure that the middleware team is aware of the lack of these endpoints definitions.

**Review by davidyuk — APPROVED** [review]

File: store/utils.js (hunk 3 above)
Comment (CedrikNikita) [reply store/utils.js:21]: There is no lack of this endpoints, the problem in that, an array in axios request on the Swagger side converting the array type of parameter inside of url request to something that mdw does not  expect `type_group[]=...`

**Pull request merged**